PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+## Your checklist for this pull request
+
+🚨 Please review the [guidelines for contributing](https://github.com/ProgramEquity/amplify/blob/david-chapuis/.github/CONTRIBUTING.md) to this repository.
+
+   - [ ] Make sure you are requesting to pull a topic/feature/bugfix branch (right side). Don't request your master!
+   - [ ] Make sure you are making a pull request against the canary branch (left side). Also you should start your branch off our canary.
+   - [ ] Check the commit's or even all commits' message styles matches our requested structure.
```

Comment:
This sounds like a good guideline but unfortunately, we don't have a 'requested structure' in the contribution guide \[yet\]. I think this guideline can be replaced with a direction to write atomic, meaningful, and clear commits.

Reply:
ok i changed the line